lists.lisp

```diff
@@ -50,6 +50,7 @@ From Emacs Lisp."
 From Emacs Lisp."
   (if (consp x) (cdr x) nil))
 
+(declaim (notinline append1))
```

Comment:
Since defsubst declaims a binding inline, this declaim is immediately overridden (same for single).

Reply:
Indeed, I didn't read this part of the code properly.

The issue is that _level1_ compiler-macros (`append1`, `single`, `keep`, `filter`) are used in _level0_.
Is the proper solution to define these compiler-macros in _level0_ and _depend-on_ it?